# nav_msgs/Path MissionPath
geographic_msgs/GeoPath MissionPath
int32 h_d
# flight angle & orientation respect to the north direction
float32 flight_angle
float32 orientation
float32 thermal_camera_shooting_distance
float32 rgb_camera_shooting_distance
---
bool MissionReceived